gen/ad_mu

```diff
@@ -518,19 +516,19 @@ for my $login (@logins) {
 
 	print FILE "c: CZ\n";
 
-	if($users->{$login}->{$A_U_F_O365_PREFERRED_LANGUAGE}) {
+	if ($users->{$login}->{$A_U_F_O365_PREFERRED_LANGUAGE}) {
 		print FILE "preferredLanguage: " . $users->{$login}->{$A_U_F_O365_PREFERRED_LANGUAGE} . "\n";
 	}
 
 	print FILE "targetaddress: " . $login . '@mail.muni.cz' . "\n";
 
-	print FILE "msDS-cloudExtensionAttribute1: " . $login . '@' . $facilityAttributes{$A_F_AZURE_DOMAIN} . "\n";
+	print FILE "msDS-cloudExtensionAttribute1: " . $login . '@' . $domain . "\n";
```

Comment:
``$domain`` contains ``A_F_DOMAIN`` and not ``A_F_AZURE_DOMAIN`` after this change. Is it ok?

Reply:
Nice catch. You are right. Fixed.

Reply:
We didn't notice because there are the same values in both attributes at this moment in testing data.